### PR TITLE
PXC-5.6-3213 Release Notes for 5.6.47-28.40

### PR DESCRIPTION
**Pull request by patrickbirch**
modified source/conf.py
modified source/release-notes/release-notes_index
added Percona-XtraDB-Cluster-5.6.47-28.40

### CODE REVIEW

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+*Percona XtraDB Cluster* 5.6.47-28.40
+================================================================================
+
+:Date: May 4, 2020
+:Installation: `Installing Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/5.6/installation.html>`_
+
+Improvements
+================================================================================
+
+* :jirabug:`PXC-3006`: Added information about xinetd 'cps' option in PXC HAProxy documentation
```
Comment (dutow) [inline doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst]: PXC-3006 is in review in jira, and affects only 5.7

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* :jirabug:`PXC-2602`: Added Ability to Configure xbstream options with wsrep_sst_xtrabackup-v2
```
Comment (dutow) [inline doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst]: Jira ticket only mentions 5.7, please also update that too.

**Review by dutow — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst]: Removed 3006

File: doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-XtraDB-Cluster-5.6.47-28.40.rst]: Added 5.6 to the ticket.

**Review by dutow — APPROVED** [review]

**Pull request merged**